tests/conftest.py

```diff
@@ -68,12 +72,12 @@ def tp(self):
 
 class MockedOtherMessage:
     """Mocks the second header returned by CQCConnection.readMessage"""
-    next_qubit_id = 0
+    next_qubit_id = 1
 
```

Comment:
The real SimulaQron backend starts with qubit ID 1, so this ensures the mock starts with qubit ID 1 as well.